README.md

```diff
@@ -155,6 +155,38 @@ var browser = require("webextension-polyfill/dist/browser-polyfill.min");
 ...
 ```
 
+### Preferred Usage with webpack
```

Comment:
How about changing the title for this section into something like "Usage with webpack without bundling"?

we should also add a link to the new section in the TOC part of this README file.